MaterialLib/Fluid/FluidProperties/FluidProperties.h

```diff
@@ -48,11 +48,16 @@ class FluidProperties
         std::unique_ptr<MaterialLib::Fluid::FluidProperty>&& heat_capacity,
         std::unique_ptr<MaterialLib::Fluid::FluidProperty>&&
             thermal_conductivity)
-    /* Will use this if MS visual studio compiler supports
-    : _property_models{{std::move(density), std::move(viscosity),
-                        std::move(heat_capacity),
-                        std::move(thermal_conductivity)}}
-    */
+#ifndef MSVC
+        // Up to the latest Visual Studio compiler, list initialization inside
```

Comment:
Added Visual Studio 2015. I did not test with the lasted msvc because I do not have it. However, I did it according to the the document on MSDN
 https://msdn.microsoft.com/en-us/library/dn793970.aspx